pr-data.csv

```diff
@@ -5340,12 +5340,12 @@ https://github.com/wildfly/wildfly,914e190167a79c538e0f87ea4385a6626bdecc9a,mars
 https://github.com/wildfly/wildfly,914e190167a79c538e0f87ea4385a6626bdecc9a,marshalling/protostream,org.wildfly.clustering.marshalling.protostream.ProtoStreamUtilTestCase.testSynchronizedMap,ID,,,Detected by Nondex 1.1.3
 https://github.com/wildfly/wildfly,914e190167a79c538e0f87ea4385a6626bdecc9a,marshalling/protostream,org.wildfly.clustering.marshalling.protostream.ProtoStreamUtilTestCase.testUnmodifiableCollection,ID,,,Detected by Nondex 1.1.3
 https://github.com/wildfly/wildfly,914e190167a79c538e0f87ea4385a6626bdecc9a,marshalling/protostream,org.wildfly.clustering.marshalling.protostream.ProtoStreamUtilTestCase.testUnmodifiableMap,ID,,,Detected by Nondex 1.1.3
-https://github.com/wildfly/wildfly,914e190167a79c538e0f87ea4385a6626bdecc9a,marshalling/spi,org.wildfly.clustering.marshalling.spi.ExternalizerUtilTestCase.testSynchronizedCollection,ID,,,Detected by Nondex 1.1.3
-https://github.com/wildfly/wildfly,914e190167a79c538e0f87ea4385a6626bdecc9a,marshalling/spi,org.wildfly.clustering.marshalling.spi.ExternalizerUtilTestCase.testSynchronizedMap,ID,,,Detected by Nondex 1.1.3
-https://github.com/wildfly/wildfly,914e190167a79c538e0f87ea4385a6626bdecc9a,marshalling/spi,org.wildfly.clustering.marshalling.spi.ExternalizerUtilTestCase.testSynchronizedSet,ID,,,Detected by Nondex 1.1.3
-https://github.com/wildfly/wildfly,914e190167a79c538e0f87ea4385a6626bdecc9a,marshalling/spi,org.wildfly.clustering.marshalling.spi.ExternalizerUtilTestCase.testUnmodifiableCollection,ID,Opened,https://github.com/wildfly/wildfly/pull/17289,Detected by Nondex 1.1.3
-https://github.com/wildfly/wildfly,914e190167a79c538e0f87ea4385a6626bdecc9a,marshalling/spi,org.wildfly.clustering.marshalling.spi.ExternalizerUtilTestCase.testUnmodifiableMap,ID,,,Detected by Nondex 1.1.3
-https://github.com/wildfly/wildfly,914e190167a79c538e0f87ea4385a6626bdecc9a,marshalling/spi,org.wildfly.clustering.marshalling.spi.ExternalizerUtilTestCase.testUnmodifiableSet,ID,,,Detected by Nondex 1.1.3
+https://github.com/wildfly/wildfly,914e190167a79c538e0f87ea4385a6626bdecc9a,marshalling/spi,org.wildfly.clustering.marshalling.spi.ExternalizerUtilTestCase.testSynchronizedCollection,ID,DeveloperFixed,https://github.com/wildfly/wildfly/pull/17292,https://issues.redhat.com/browse/WFLY-18639
```

Comment:
Add in the notes (last column) a link to the commit that fixed the test (not a link to some issue). Let us know via email or a private Campuswire post where the logs are showing that the tests fail before this commit and pass after the commit.